services/comm/db/seeds/development/data.seeds.rb

```diff
@@ -5,9 +5,9 @@
     # next if tenant.id.eql? 1
     tenant.switch do
       Providers::Twilio.create(name: "Marketing Team's Twilio", account_sid: ENV['TWILIO_ACCOUNT_SID'],
-                               auth_token: ENV['TWILIO_AUTH_TOKEN'])
```

Comment:
I'd then argue that this should only be created for the platform owner (first tenant)